internal/alb/tg/targetgroup.go

```diff
@@ -132,7 +136,7 @@ func NewDesiredTargetGroupFromBackend(o *NewDesiredTargetGroupFromBackendOptions
 		CommonTags:     o.CommonTags,
 		Store:          o.Store,
 		LoadBalancerID: o.LoadBalancerID,
-		TargetPort:     port,
+		TargetPort:     targetGroupDefaultPort,
```

Comment:
Since this is effectively a hard coded default now, can we remove this parameter completely and put the port in `NewDesiredTargetGroupFromBackend`?

Reply:
Yes, it can be removed safely since we won't change the value.

I'll create another PR to fixed as we talked in slack 😄